core/modules/ip_lookup.cc

```diff
@@ -302,6 +304,47 @@ CommandResponse IPLookup::CommandAdd(
   return CommandSuccess();
 }
 
+CommandResponse IPLookup::CommandDelete(
+    const bess::pb::IPLookupCommandDeleteArg &arg) {
+  using bess::utils::be32_t;
+
+  be32_t net_addr;
+  be32_t net_mask;
+
+  if (!arg.prefix().length()) {
+    return CommandFailure(EINVAL, "prefix' is missing");
+  }
+  if (!bess::utils::ParseIpv4Address(arg.prefix(), &net_addr)) {
+    return CommandFailure(EINVAL, "Invalid IP prefix: %s",
+                          arg.prefix().c_str());
+  }
+
+  uint64_t prefix_len = arg.prefix_len();
+  if (prefix_len > 32) {
+    return CommandFailure(EINVAL, "Invalid prefix length: %" PRIu64,
+                          prefix_len);
+  }
+
+  net_mask = be32_t(~((1ull << (32 - prefix_len)) - 1));
```

Comment:
nit: Could you use the [SetBitsLow()](https://github.com/NetSys/bess/blob/master/core/utils/bits.h#L193) utility here instead? There's an example of its usage with the big-endian types [here](https://github.com/NetSys/bess/blob/master/core/utils/ip.cc#L78)